doc/getting-started/apl-nuc.rst

```diff
@@ -325,18 +312,18 @@ Set up Reference UOS
 
    .. code-block:: none
 
-      # unxz clear-26200-kvm.img.xz
+      $ unxz clear-26200-kvm.img.xz
 
 #. Deploy the UOS kernel modules to UOS virtual disk image (note: you'll need to use
-   the same **standard** image version number noted in step 1 above):
+   the same **iot-lts2018** image version number noted in step 1 above):
 
    .. code-block:: none
 
-      # losetup -f -P --show /root/clear-26200-kvm.img
-      # mount /dev/loop0p3 /mnt
-      # cp -r /usr/lib/modules/4.19.0-19.iot-lts2018 /mnt/lib/modules/
-      # umount /mnt
-      # sync
+      $ sudo losetup -f -P --show clear-26200-kvm.img
+      $ sudo mount /dev/loop0p3 /mnt
```

Comment:
That reminds me that we never instruct the user to `umount /mnt` after we have modified the ESP to add the `acrn.efi` and `acrn.conf`. Can you add that instruction above?